### ISSUE TITLE
something wrong at the attack table in online multiplayer mode

### ISSUE THREAD

**Issue by SundayLee109** [MrStahlfelge/lightblocks#41]
the currently attack table is:
single 0
double 1
triple 2
tetris 4
TSS 0
TSD 4
TST 4
PC 0?
B2B 0
combo 0
use classic type with this table is ok, but not really with modern type
should change thing is:
TSS 2
TST 6
B2B 1(at least)
PC 10
combo need to have attack bonus

**Comment by SundayLee109** [comment]
another one is the ping is pretty hurt
when I press the button, need to wait for a long time to let it work if ping is too high, sometimes it won't work.

**Comment by MrStahlfelge** [comment]
> another one is that the ping is pretty hurt
> when I press the button, need to wait for a long time to let it work if ping is too high, sometimes it won't work.

You are right about that one. We have a good performing European server and a bad performing American server, so everyone outside Europe won't get to play the game good. You can run the server on your local network and check how good it could be. 
I am stuck here and had hoped for the community to step in, as I am not a profit oriented company I can't afford to provide servers for a good connection around the globe. :) I am not sure how the websites bootstrapped their service, I have a bootstrapping problem because of the problems I don't let the feature go out of beta, which of course means no one uses it, which means no one is willing to provide servers...

**Comment by Yonle** [comment]
If you're afraid of bots/cheating issue, Probably you could add Anti-cheat triggers at the server.
